ci-operator/step-registry/ipi/install/post-install/ipi-install-post-install-commands.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+
+# if the cluster profile includes an insights secret, install it to the cluster to
+# report support data from the support-operator
+if [[ -f "${CLUSTER_PROFILE_DIR}/insights-live.yaml" ]]; then
```

Comment:
I don't think this should be conditional. Either a test depends on this being present, and this should fail if it does not succeed, or it should just not run this step.

Reply:
Yea, I made this conditional, as I described in the previous comment, because I wanted to make this step to straightfull related with that secret.